doc/api/http2.md

```diff
@@ -159,18 +159,16 @@ an `Http2Session`.
 added: v8.4.0
 -->
 
+* `type` {Integer} An integer identifying the frame type.
```

Comment:
It seems all three should be `{integer}` due to [type-parser.js](https://github.com/nodejs/node/blob/0f8caf23bed91cbbb19121a3c97bb21c703258ac/tools/doc/type-parser.js#L8)

Reply:
Nit: You don't need to tell the reader it's an integer if it already has the type `{integer}`. So instead of everything starting with `{integer} An integer identifying the frame type.`, just say `{integer} The frame type.`